libs/api/domains/rsk-company-info/src/lib/graphql/rsk-company-info.resolver.ts

```diff
@@ -10,7 +9,7 @@ import { RskCompanySearchItems } from './models/rskCompanySearchItems.model'
 import { LOGGER_PROVIDER } from '@island.is/logging'
 import type { Logger } from '@island.is/logging'
 
-@UseGuards(IdsUserGuard, ScopesGuard)
+// @UseGuards(IdsUserGuard, ScopesGuard)
```

Comment:
Should these resolvers and all returned data be publicly accessible?

Reply:
@eirikurn nope my bad here. Was commented out for testing locally, should not have been included in the commit